js/ai/src/embedder.ts

```diff
@@ -18,32 +18,57 @@ import { Action, defineAction, z } from '@genkit-ai/core';
 import { Registry } from '@genkit-ai/core/registry';
 import { Document, DocumentData, DocumentDataSchema } from './document.js';
 
+/**
+ * A batch (array) of embeddings.
+ */
 export type EmbeddingBatch = { embedding: number[] }[];
 
+/**
+ * Zod schema of an embedding vector.
+ */
```

Comment:
This one is not used anywhere else, does it still need to be `export`ed and provided a JSDoc? 

I reckon there will be more that are in similar state, should we cleanup vs exporting them?

Reply:
used in EmbedResponseSchema

Reply:
Yes, but `EmbedResponseSchema` is in this same file though,

`EmbeddingSchema`  is exported in `js/genkit/src/embedder.ts` which is public API so un-exporting this will be a breaking change.

Reply:
The `Embedding` type is also used as return type of `embed` function. I got rid of `Embedding` though.